bears/general/CLOCBear.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises file of or number of files in directory structure
```

Comment:
Will correct it. And regarding previous comment, `__init__.py` is required. I had pushed a commit without that file, and it failed on CI. I can't recall which bear checks that, but I believe, that if there's python file in a folder, then `__init__.py` file is required.